cnd/src/http_api/problem.rs

```diff
@@ -50,6 +51,10 @@ pub fn from_anyhow(e: anyhow::Error) -> HttpApiProblem {
         return HttpApiProblem::new("Swap not found.").set_status(StatusCode::NOT_FOUND);
     }
 
+    if let Some(e) = e.downcast_ref::<SwapExists>() {
+        return HttpApiProblem::new(format!("{}", e)).set_status(StatusCode::BAD_REQUEST);
```

Comment:
I would prefer user-facing errors to be static strings. It is not a breaking change per-se but to me it would be very surprising that changing the error message of an error defined 3 abstraction levels deep has a user-facing impact.

Also, the error message here is not consistent with the others as it is:

- not capitalized
- doesn't end with a dot

Reply:
In the other handlers, we log the error :)

Reply:
It might also be worth to tell the user a little more of what is going here and they just need to wait until the first swap finalizes before they can make a new one. Esp. the temporary nature of this error is something that should be pointed out IMO.

Reply:
Cool, I didn't understand this on first reading but after looking at the code again I've got you.  Cheers man, will fix up this PR as suggested.